sim/druid/items.go

```diff
@@ -197,14 +197,61 @@ var ItemSetDeepEarthRegalia = core.NewItemSet(core.ItemSet{
 	Bonuses: map[int32]core.ApplyEffect{
 		// Insect Swarm increases all damage done by your Starfire, Starsurge, and Wrath spells against that target by 3%
 		2: func(agent core.Agent) {
+			const (
+				DDBC_2pcT13 int = iota
+
+				DDBC_Total
+			)
```

Comment:
These constants can likewise be removed if you set `DamageDoneByCasterMultiplier` for each `AttackTable` directly. Furthermore, even in the event that additional `DynamicDamageDoneByCaster` modifiers need to be added for the class down the road, the indexing constants supplied to `EnableDamageDoneByCaster()` need to be defined at the top level of the `druid` package so that they can be accessed by all aura implementations that need them (similar to the class spell mask constants).